docroot/modules/custom/foia_webform/src/FileFieldLookupServiceInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\foia_webform;
+
+use Drupal\webform\WebformInterface;
+
+/**
+ * Provides interface defining the File Field Lookup Service.
+ */
+interface FileFieldLookupServiceInterface {
+
+  /**
+   * Gets the machine names of all file attachment elements on the webform.
+   *
+   * @param \Drupal\webform\WebformInterface $webform
+   *   The webform being submitted against.
+   *
+   * @return array
+   *   Returns an array of machine names of file attachment elements on the
+   *   webform being submitted against.
+   */
+  public function getFileAttachmentElementsOnWebform(WebformInterface $webform);
```

Comment:
Similarly, if we want to keep this, it seems better if it were protected rather than public.

Reply:
That function needs to be called from other classes, so it needs to be public, unless we add a different public method that calls that method.